web/src/main/webapp/WEB-INF/pages/application_state.jsp

```diff
@@ -6,26 +6,246 @@
-            <div id="state_table">
-                <div>
-                    <table class="table table-hover" id="states_table"  data-toggle="table" data-url="../secure/applications_states/${applicationId}" data-height="500"
-                    data-sort-name="name">
-                        <thead>
-                            <tr>
-                                <th data-field="stateType" data-align="left"><spring:message
-                                                code="application.table.state"/></th>
-                                <th data-field="creationDate" data-align="left"><spring:message
-                                                code="application.table.creationdate"/></th>
-                                <th data-field="description" data-align="left"><spring:message
-                                                code="application.table.description"/></th>
-                            </tr>
-                        </thead>
-                    </table>
-                </div>
-            </div>
-        </div>
-    </jsp:body>
+    <jsp:attribute name="pageJs">
+        <script src="<c:url value="/resources/js/atsy-statehistory.js" />"></script>
+    </jsp:attribute>
+  <jsp:body>
+      <div class="page-header">
+          <h1><spring:message code="application.state.title"/>
+              <small id="positionName">${states[0].position.name}</small>
+          </h1>
+      </div>
+      <spring:message code="candidate.table.state.newstate" var="newstate" />
+      <spring:message code="candidate.table.state.cv" var="cv" />
+      <spring:message code="candidate.table.state.hr" var="hr" />
+      <spring:message code="candidate.table.state.firstTest" var="firstTest" />
+      <spring:message code="candidate.table.state.tech" var="tech" />
+      <spring:message code="candidate.table.state.coding" var="coding" />
+      <spring:message code="candidate.table.state.profInterview" var="profInterview" />
+      <spring:message code="candidate.table.state.clientInterview" var="clientInterview" />
+      <spring:message code="candidate.table.state.wageOffer" var="wageOffer" />
+      <spring:message code="candidate.table.state.agree" var="agree" />
+      <spring:message code="candidate.table.state.accept" var="accept" />
+      <spring:message code="candidate.table.state.reject" var="reject" />
+      <spring:message code="candidate.table.state.pause" var="pause" />
+      <div class="button-panel clearfix">
+          <form class="form-inline pull-right">
+              <c:if test="${states[0].stateDTO.name != reject && states[0].stateDTO.name != accept}">
+              <div class="form-group">
+                  <label class="form-label">Következő lépés:</label>
+              </div>
+              <div class="form-group">
+                  <div class="btn-group" role="group">
+                      <c:choose>
+                          <c:when test="${states[0].stateDTO.name == newstate}">
```

Comment:
Don't use the localized message to find out which section should be displayed. Use the internal name of the state instead.